site/content/docs/5.3/content/tables.md

```diff
@@ -906,10 +906,7 @@ Use SVG to display icons or thumbnails in your compact table data cell elements
             </div>
           </td>
           <td>
-            <svg xmlns="http://www.w3.org/2000/svg" width="1.875rem" height="1.875rem" class="me-1" preserveAspectRatio="xMidYMid slice" focusable="false" role="img" aria-labelledby="svg1">
-              <title id="svg1">Thumbnail</title>
-              <rect width="100%" height="100%" fill="#ffd200"></rect>
-            </svg>
+            <img src="/docs/{{< param docs_version >}}/assets/img/thumbnail.png" alt="Thumbnail" width="30" height="30">
```

Comment:
In the introduction of this section, it's said:

> Use SVG to display icons or thumbnails in your compact table data cell elements[...]

This image is not an SVG. Do we change the sentence to say "SVG or PNG"?

Moreover, didn't we say that we would use only the thumbnail image for each row (removing the document image)? In this case, it might be worth changing the `thumbnail.png` to an SVG to stick with the original statement of this section OR changing the sentence to talk about "PNG", "image", or something else.

Reply:
Discussed with Franco and the team in the weekly sync, The thumbnail is ok (same than in the DSM) and it is ok to have both in the same table... Introduction text should be adapted though, to reflect the possibilities and the code.